src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -81,44 +81,44 @@ public static Address parseAddress(String address) throws ParseException {
     }
 
     /**
-     * Parses a {@code String email} into an {@code Email}.
+     * Parses a {@code String email} into an {@code Date}.
      * Leading and trailing whitespaces will be trimmed.
      *
      * @throws ParseException if the given {@code email} is invalid.
      */
-    public static Email parseEmail(String email) throws ParseException {
+    public static Date parseEmail(String email) throws ParseException {
```

Comment:
`parseEmail` should be renamed to `parseDate` with the corresponding parameter renamed. Same for all other instances.